### ISSUE TITLE
Size mismatch of converted TF Saved Model to tfjs model

### ISSUE THREAD

**Issue by 11mhg** [tensorflow/tfjs#3695]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version

{ 'tfjs-core': '2.0.1',
  'tfjs-backend-cpu': '2.0.1',
  'tfjs-backend-webgl': '2.0.1',
  'tfjs-data': '2.0.1',
  'tfjs-layers': '2.0.1',
  'tfjs-converter': '2.0.1',
  tfjs: '2.0.1',
  'tfjs-node': '2.0.1' }

#### Browser version

N/A, Using Node

#### Describe the problem or feature request

I have some model in tensorflow python which I have trained and saved as a TF Saved Model using the following line of code:
```
    tf.saved_model.save(model, './saved_models/REPainter',
                        signatures=model.call.get_concrete_function(
                            tf.TensorSpec(shape=[None,None,None,6], dtype=tf.float32)))
```
I can then load the model in python and predict using an arbitrary input size and it works fine in python:
```
model.signatures['serving_default'](tf.ones([1,416,416,6]))['output_0'].shape
```
However, once I convert to a tfjs model using this call:
```
tensorflowjs_converter --input_format=tf_saved_model --signature_name=serving_default --output_node_names=output_0 ./saved_models/REPainter ./tfjs_model/
```
My tfjs model cannot run. It seems as though it is infering incorrect shapes. To be specific, the following happens:

![image](https://user-images.githubusercontent.com/9386263/88939004-03240700-d254-11ea-8e86-a5bd2af544eb.png)

I have no clue what is causing this. Any clue?

**Comment by pyu10055** [comment]
This could be related to strided_slice op, we have a fix for that in upcoming 2.1.0 release.

**Comment by 11mhg** [comment]
Sounds good. I will give this a go with 2.1.0. Is there a dev preview or smth I can use to try it out @pyu10055 ?
